tests/test_setup.py

```diff
@@ -0,0 +1,304 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+"""
+Usage:
+This file defines tasks for building Protos, webhost and extensions
+
+To use these tasks, you can run the following commands:
+
+1. Build protos:
+   invoke -c test_setup build_protos

```

Comment:
Same comment about build_protos vs build-protos. This also requires `cd tests` to be run first, correct? (eg. you can't run  `invoke -c test_setup build_protos` from the root directory)

Reply:
Fixed.